src/index.js

```diff
@@ -27,9 +28,18 @@ function getPublicIp(callback) {
   });
 }
 
-function getDnsRecords(domain, host, callback) {
-  const path = `/v1/domains/${domain}/records/A/${host}`;
-  requestApi('GET', path, null, true, callback);
+function getDnsRecords(domain, hosts, callback) {
+  async.map(hosts, function (host, callback) {
+    const path = `/v1/domains/${domain}/records/A/${host}`;
+    requestApi('GET', path, null, true, (err, response) => {
+      if (err){
+        return;
+      }
```

Comment:
That works too. Return void just makes sure it's not actually returning anything.